frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -274,6 +288,82 @@ export default {
     getAddUserSuccessMsg() {
       return ADD_USER_SUCCESS_MSG
     },
+
+    /**
+     * Returns true if users facility access has changed, false otherwise.
+     */
+    hasUserFacilityAccessChanged(selectedFacilityIds, userFacilities) {
+      if (selectedFacilityIds?.length !== userFacilities?.length) {
+        return true;
+      } else {
+        for (let facilityId of selectedFacilityIds) {
```

Comment:
I think this loop can be simplified. Something like https://stackoverflow.com/questions/16312528/check-if-an-array-contains-any-element-of-another-array-in-javascript.

Reply:
Were you able to look at a simpler solution?

Reply:
sorry, missed this... addressed with .some .includes.